src/components/aeIdentity/aeIdentity.js

```diff
@@ -30,7 +26,7 @@ export default {
     */
     active: {
       type: Boolean,
-      default: true
```

Comment:
Why change it?

Reply:
I think at best within the app, most cards are supposed to be inactive except for one. So it would be best to set the default to *false* rather than true for every case.

Reply:
I won't accept changing this because of several reasons:

1) This change is not related to the rest of the changes in this PR. If we want to have a discussion about this, it becomes blurred and mixed with the rest of the changes. It blocks our progress. 

2) It's an arbitrary change that you decided to make based on a personal preference. It could potentially break others people code that don't expect such arbitrary change of default behaviour. Also it could inspire other devs to demand similar changes and distract what is really important.

3) I feel like `inactive` identity is the rarer (more special, thus requires more 'work') use case.
for example; html input element is also not 'disabled' by default.

4) Changing default behaviour is not creating any progress. It will not make anybodies experience with the components significantly easier to understand or faster to develop. 


I understand, this is just a tiny change and it might seem like I'm overly strict and conscious. However I believe that we have to start **training and enforcing a good workflow**, like separation of concerns, **right away**. As complexity grows rapidly, we would simply miss the moment, when it will be of crucial importance. Also our PRs should be a best practice example for other developers.